src/common/mfu_flist_copy.c

```diff
@@ -150,6 +150,33 @@ static void mkdir_progress_fn(const uint64_t* vals, int count, int complete, int
 static mfu_copy_file_cache_t mfu_copy_src_cache;
 static mfu_copy_file_cache_t mfu_copy_dst_cache;
 
+/* close a file that opened with mfu_copy_open_file */
+
+static int mfu_copy_close_file(
```

Comment:
(style) Why move this?  It doesn't hurt anything, but it makes it much harder to see if there are changes and so harder to see if a bug was created accidentally.

Reply:
I'll move it back and add a prototype at the beginning of the file.

Reply:
I'll add a new MR for that one as it is a different issue.